src/identifier.h

```diff
@@ -21,12 +21,18 @@
 
 class Identifier : public RootObject
 {
+private:
```

Comment:
I had removed these at some point because they are private data and the C++ code should never access them.